README.md

```diff
@@ -67,10 +67,12 @@ For crispr screening:
    - ([`bowtie2`](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml))
 3. Optional: CNV correction and normalization with ([`CRISPRcleanR`](https://github.com/francescojm/CRISPRcleanR))
 4. Rank sgRNAs and genes ;
-   a. ([MAGeCK test](https://sourceforge.net/p/mageck/wiki/usage/#test))
-   b. ([MAGeCK mle](https://sourceforge.net/p/mageck/wiki/Home/#mle))
-   c. ([BAGEL2](https://github.com/hart-lab/bagel))
-5. Visualize analysis
+   - ([MAGeCK test](https://sourceforge.net/p/mageck/wiki/usage/#test))
+   - ([MAGeCK mle](https://sourceforge.net/p/mageck/wiki/Home/#mle))
+   - ([BAGEL2](https://github.com/hart-lab/bagel))
+   - ([DrugZ](https://github.com/hart-lab/drugz))
```

Comment:
Should this go under point 5?

Reply:
Actually no it's also a gene essentiality analysis but only on drug treated cell lines. It does something similar to MAGeCK MLE